CRM/Admin/Form/PaymentProcessor.php

```diff
@@ -486,7 +476,9 @@ public function updatePaymentProcessor(&$values, $domainID, $test) {
       'accepted_credit_cards' => $creditCards,
     ], $values);
 
-    civicrm_api3('PaymentProcessor', 'create', $params);
+    $result = civicrm_api3('PaymentProcessor', 'create', $params);
```

Comment:
Might as well switch to use v4 since it will get r-run testing which is the hardest part of switching :-)

Reply:
Ok done